src/dev/src/app/quick-list/quick-list.demo.html

```diff
@@ -2,15 +2,18 @@ <h2>Quick List</h2>
 <div class="clr-example">
   <div class="main-container">
     <div class="clr-row">
-      <clr-quick-list
-        class="clr-col-3"
-        [clrAllValues]="possibleActivityResults"
-        [(clrValues)]="selectedActivityResults"
-        [clrAddLabel]="getAddLabel()"
-        [clrBlankOption]="BLANK_OPTION"
-        [clrMandatory]="true"
-      >
-      </clr-quick-list>
+      <form clrForm class="clr-row">
+        <clr-quick-list
+          class="clr-form-control clr-col-12 clr-row"
+          clrControlClasses="clr-col-lg-3 clr-col-md-4 clr-col-sm-5"
+          [clrAllValues]="possibleActivityResults"
+          [(clrValues)]="selectedActivityResults"
+          [clrAddLabel]="getAddLabel()"
+          [clrBlankOption]="BLANK_OPTION"
+        >
+          <label class="clr-control-label  clr-col-md-2 clr-col-sm-3 required">Option list</label>
+        </clr-quick-list>
+      </form>
```

Comment:
just add a clr-input-container (like in combobox demo) to check if the layout inside a form is aligned with the standard elements